src/dowel/csv_output.py

```diff
@@ -15,6 +16,8 @@ class CsvOutput(FileOutput):
 
     def __init__(self, file_name):
         super().__init__(file_name)
+        self._file_name = file_name
+        self._file_sections = [file_name+'_0']
```

Comment:
This concept of "file sections" seems a little complex (although it probably have benefits over a simpler design). It would be good to have a comment describing what the advantage is of doing it this way instead of another design.

It's also not completely clear that a list is really needed here. Under which circumstances are there more than two sections?

Reply:
The benefit is updated in https://github.com/SuperElephant/dowel/pull/1#issue-420458246.

Actually, "file sections" is a "redundant" design that can support more customized situations in manual logging. It provides some forward compatibility (file might be splited in different sections for other reasons and needs merge afterwards), while not reduce the efficiency.